setup.cfg

```diff
@@ -1,17 +1,23 @@
 [pycodestyle]
+max-line-length = 88
 exclude = .eggs,ENV,build,docs/conf.py,venv
+add-ignore = D105
 
 [yala]
 radon mi args = --min C
 # TODO later fix raise-missing-from
-pylint args = --disable=too-few-public-methods,too-many-instance-attributes,raise-missing-from,use-dict-literal,unnecessary-pass --ignored-modules=napps.kytos.mef_eline
-linters=pylint,pycodestyle,isort
+pylint args = --disable=no-name-in-module,too-many-locals,too-few-public-methods,too-many-instance-attributes,no-else-return,dangerous-default-value,duplicate-code,raise-missing-from,too-many-arguments,too-many-public-methods,unnecessary-pass,import-error,relative-beyond-top-level,attribute-defined-outside-init --ignored-modules=napps.kytos.mef_eline
```

Comment:
`napps.kytos.mef_eline` shouldn't be needed here (even though it had already been in this file), also let's try to keep the same number of excluded args, looks like way more have been added, if you review this part again. Thanks, Heriberto. Other than that, it's looking great.

Reply:
I removed the ignored modules and reduced the number of excluded args to what it was before.